core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -29,28 +29,22 @@
 import io.grpc.LoadBalancer.PickResult;
 import io.grpc.LoadBalancer.PickSubchannelArgs;
 import io.grpc.LoadBalancer.SubchannelPicker;
-import io.grpc.PickFirstBalancerFactory;
+import io.grpc.LoadBalancerProvider;
+import io.grpc.LoadBalancerRegistry;
 import io.grpc.Status;
-import java.lang.reflect.Method;
 import java.util.List;
-import java.util.Locale;
 import java.util.Map;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
 final class AutoConfiguredLoadBalancerFactory extends LoadBalancer.Factory {
-
-  @VisibleForTesting
-  static final String ROUND_ROBIN_LOAD_BALANCER_FACTORY_NAME =
-      "io.grpc.util.RoundRobinLoadBalancerFactory";
-  @VisibleForTesting
-  static final String GRPCLB_LOAD_BALANCER_FACTORY_NAME =
-      "io.grpc.grpclb.GrpclbLoadBalancerFactory";
+  private static final String DEFAULT_POLICY = "pick_first";
```

Comment:
I don't feel that way. I name it "DEFAULT_POLICY" here so that the rest of the code can refer to it as the "default" policy without bothering what the default is. It's just another level of indirection, and doesn't imply it could be changed or not.